src/starlite_saqlalchemy/service.py

```diff
@@ -125,6 +124,8 @@ async def enqueue_background_task(self, method_name: str, **kwargs: Any) -> None
             method_name: Method on the service object that should be called by the async worker.
             **kwargs: Arguments to be passed to the method when called. Must be JSON serializable.
         """
+        from starlite_saqlalchemy.worker import queue  # pylint: disable=C0415
```

Comment:
> So at this point, if a user calls the `enqueue_background_task()` error without the SAQ extra installed, this will raise an import error which will translate to a `500`, correct?

Yes. I suppose we should try/except it to raise something nicer?

Reply:
> From a quick look at the [import system docs](https://docs.python.org/3/reference/import.html#the-module-cache), It appears that modules imported only once, and stored in `sys.modules` for future use, so if `worker` was previously imported, the next import will basically result in an attribute access to `worker.queue`.

Yep sweet, this works then. It's a method I'd frequently make a blocking call on in context of handling a request, so just want to ensure it is as efficient as possible.